src/_companies/europe/united-kingdom/london/easol/index.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Easol
+logo: logo.png
+
+continent: Europe
+country: United Kingdom
+region: London
+city: London
+
+website: https://easol.com
+careers_page: https://easol.com/careers
+
+work_arrangements:
+- onsite
+
+stack:
+- rails
+- postgres
+- stimulus
+
+created_at: 2023-03-21 13:12:11
+---
+
+Easol, founded in 2017, is the all-in-one platform for managing your entire
+experience business, on your terms. Sell directly on your own website with smart
+booking, payment, and marketing tools that all work together, brilliantly.
+
+Creating experiences is the best job in the world! But it hasn't always been
+that way - because experience creators have been suffering with the wrong tools.
+With Easol, there is no more stitching together ineffective tools built for
+selling "things", paying third-party booking fees or having to create pricey
+custom solutions.
+
+We believe that with the right tools, anyone can turn their passion into an
```

Comment:
Could you change this to be in the third person? Then it will be consistent with all other company descriptions :)